iommi/form__tests.py

```diff
@@ -3579,3 +3579,49 @@ def test_bootstrap_crash_when_no_iommi_name_on_field_group(settings):
         fields__name__group='group',
         iommi_style='bootstrap',
     ).bind(request=req('get')).render_to_response()
+
+
+def test_render_fields_template():
+    class MyForm(Form):
+        class Meta:
+            fields_template = Template('''
+            <div class="row">
+                <div class="col">
```

Comment:
We can loop over fields instead of hardcoding the two names. That's a nicer example.

Reply:
in my bs5 project I use it for 1-2 fields per row, depends on whether it's input|select or textarea and I also use bs cards (with headers) for "fieldsets", so hardcoding seems to me more like a RL example than a loop, but it's up to you

Reply:
I changed it to a more RL example, or do you still want a loop?